packages/database/src/migrations/20201123061122-AddConfirmedWeeklyDataReportForPSSS-modifies-data.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT_CODE = 'PSSS_CONFIRMED_WEEKLY_REPORT';
```

Comment:
Perhaps we want to use "first letter upper" snake case for report codes? `PSSS_Confirmed_Weekly_Report`, similar to existing `dashboardReport` ids